src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/ccd/Nationality.java

```diff
@@ -0,0 +1,274 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ccd;
+
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum Nationality {
```

Comment:
This is already implemented in RIA-3628, which will be merged to master. Rebase on master should cover this.  No change required

Reply:
same as above